web/src/compose_recipient.ts

```diff
@@ -37,24 +37,14 @@ function composing_to_current_topic_narrow(): boolean {
 }
 
 function composing_to_current_private_message_narrow(): boolean {
```

Comment:
We could simplify the check here now that we are only comparing ids. Since sorting them doesn't matter, we can just check if their intersection is of the same length as them.

Reply:
Sure, updated!